packages/eslint-config-hop-protocol/index.js

```diff
@@ -0,0 +1,114 @@
+module.exports = {
+  "env": {
+    "node": true,
+    "browser": true,
+    "es2021": true,
+    "jest": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:json/recommended",
+    // "plugin:@typescript-eslint/recommended-type-checked", // this is causing issues with some packages (core, stats-worker). Need to debug and re-enable
+    "standard"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2021,
+    "sourceType": "module",
+    "project": "./tsconfig.json"
+  },
+  "plugins": [
+    "@typescript-eslint",
+    "node",
+    "unused-imports",
+    "sort-imports-es6-autofix"
+  ],
+  "rules": {
+    // Explicit offs
+    "@typescript-eslint/consistent-type-definitions": 0,
+    "@typescript-eslint/explicit-function-return-type": 0,
+    "@typescript-eslint/method-signature-style": 0,
+    "@typescript-eslint/no-dynamic-delete": 0,
+    "@typescript-eslint/no-misused-promises": 0,
+    "@typescript-eslint/no-non-null-assertion": 0,
+    "@typescript-eslint/no-unsafe-enum-comparison": 0,
+    "@typescript-eslint/require-await": 0,
+    "@typescript-eslint/restrict-template-expressions": 0,
+    "@typescript-eslint/strict-boolean-expressions": 0,
+    "arrow-body-style": 0,
+    "avoidEscape": 0,
+    "camelcase": 0,
+    "comma-dangle": 0,
+    "indent": 0,
+    "multiline-ternary": 0,
+    "no-async-promise-executor": 0,
```

Comment:
IMO rule should be followed (ignore this suggestion if too much breaks).

```suggestion
```

Reply:
Resolved with https://github.com/hop-protocol/hop/pull/545/commits/19782a3ed6c46210c7cd1fea8dc39b26806c93d9